pkg/networkmanager/interfaces.js

```diff
@@ -3107,6 +3097,71 @@ function PageNetworkVlanSettings() {
     this._init();
 }
 
-shell.dialogs.push(new PageNetworkVlanSettings());
+function dialog_setup(d) {
```

Comment:
This section needs a comment explaining that this is porting old code, in order to explain the unexpected code that follows.